api/soap/mantisconnect.wsdl

```diff
@@ -288,6 +330,19 @@
    <xsd:element name="require_closed" type="xsd:boolean" minOccurs="0"/>
   </xsd:all>
  </xsd:complexType>
+ <xsd:complexType name="FilterCustomField">
+  <xsd:all>
+   <xsd:element name="field" type="tns:ObjectRef" minOccurs="1"/>
+   <xsd:element name="value" type="tns:StringArray" minOccurs="1"/>
```

Comment:
IMHO is better to have an opportunity to search by more custom field values at the same time.